src/main/java/org/sagebionetworks/web/client/context/KeyFactoryProvider.java

```diff
@@ -0,0 +1,7 @@
+package org.sagebionetworks.web.client.context;
+
+import org.sagebionetworks.web.client.jsinterop.KeyFactory;
+
+public interface KeyFactoryProvider {
+  public KeyFactory getKeyFactory(String currentUserAccessToken);
+}
```

Comment:
Create a mockable/injectable KeyFactoryProvider so I can easily mock KeyFactory in tests that don't support JsInterop